src/account.h

```diff
@@ -103,7 +103,8 @@ class account_t : public flags::supports_flags<>, public scope_t
   virtual ~account_t();
 
   virtual string description() {
-    return string(_("account ")) + fullname();
+    // TRANSLATORS: %1% refers to the full name of the account
```

Comment:
These special `TRANSLATORS` comments show up in the `.po` files and provide context for translators.